### PR TITLE
VEN-491 | VEN-492 | VEN-493 | Customer UI fixes

### PR DESCRIPTION
**Pull request by santtul**
### Description

- Add .nvmrc
- Improve readme
- Adjust unregistered boat input and step name text colors
- Move focus to page start when moving between steps

### Resolves
- https://helsinkisolutionoffice.atlassian.net/browse/VEN-491
- https://helsinkisolutionoffice.atlassian.net/browse/VEN-492
- https://helsinkisolutionoffice.atlassian.net/browse/VEN-493

### CODE REVIEW

File: src/components/forms/fragments/UnRegisteredBoatDetails.tsx [hunk 1]
```diff
@@ -34,7 +34,11 @@ const UnRegisteredBoatDetailsFragment = ({
       <Row>
         {boatTypes && (
           <Col sm={4}>
-            <BoatType boatTypes={boatTypes} required={!fieldsNotRequired} />
+            <BoatType
+              boatTypes={boatTypes}
+              required={!fieldsNotRequired}
+              className="unregistered-boat-input"
```
Comment (Nizar-Rahme) [inline src/components/forms/fragments/UnRegisteredBoatDetails.tsx:40]: In the customer UI, we're following BEM naming convention, prefixed with `vene`
so this className would be something like: `vene-unregistered-boat__input`

File: src/components/forms/fragments/UnRegisteredBoatDetails.tsx [hunk 2]
```diff
@@ -50,6 +54,7 @@ const UnRegisteredBoatDetailsFragment = ({
             append="m"
             min="0"
             required={!fieldsNotRequired}
+            className="unregistered-boat-input"
```
Comment (Nizar-Rahme) [inline src/components/forms/fragments/UnRegisteredBoatDetails.tsx:57]: And here.

File: src/components/forms/fragments/UnRegisteredBoatDetails.tsx [hunk 3]
```diff
@@ -65,6 +70,7 @@ const UnRegisteredBoatDetailsFragment = ({
             append={boatStoredOnTrailer ? '+1 m' : 'm'}
             min="0"
             required={!fieldsNotRequired}
+            className="unregistered-boat-input"
```
Comment (Nizar-Rahme) [inline src/components/forms/fragments/UnRegisteredBoatDetails.tsx:73]: ^

File: src/components/forms/Wizard.tsx [hunk 4]
```diff
@@ -58,12 +59,22 @@ class Wizard extends Component<Props, State> {
...
   };
 
+  focusFirstPageElement = () => {
+    const mainLink = document.getElementById('main-link');
+    if (mainLink) {
+      mainLink.focus();
+      // Lose the focus so that blue rect is not displayed
+      mainLink.blur();
+    }
+  };
```
[the remark this answers is not in the document]
Comment (santtul) [reply src/components/forms/Wizard.tsx:76]: I think you can only create refs for child components?

File: src/components/forms/fragments/UnRegisteredBoatDetails.tsx (hunk 1 above)
Comment (santtul) [reply src/components/forms/fragments/UnRegisteredBoatDetails.tsx:40]: Will fix.

File: src/components/forms/Wizard.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (Nizar-Rahme) [reply src/components/forms/Wizard.tsx:76]: Could https://reactjs.org/docs/forwarding-refs.html be in use here?

File: src/components/forms/Wizard.tsx [hunk 5]
```diff
@@ -58,12 +59,22 @@ class Wizard extends Component<Props, State> {
...
       goBackwards(values);
     }
   };
 
+  focusFirstPageElement = () => {
+    const mainLink = document.getElementById('main-link');
+    if (mainLink) {
+      mainLink.focus();
+      // Lose the focus so that blue rect is not displayed
+      mainLink.blur();
```
[the remark this answers is not in the document]
Comment (santtul) [reply src/components/forms/Wizard.tsx]: Done.

File: src/components/forms/Wizard.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (santtul) [reply src/components/forms/Wizard.tsx:74]: Agreed that it would be a lot more complex with passing refs around, so keeping the current.

**Pull request merged**